### PR TITLE
testing/handlebars: new aport, Handlebars C: libs, dev and utils

### PR DESCRIPTION
**Pull request by rvalle**
Hi!

This is a port of Handlebars.js to C.

The purpose of including this package in Alpine is double:

Library for providing native bindings for handlebars templates into other languages (PHP, etc).
Lightweight and efficient template engine utilities.

For details see: https://github.com/jbboehr/handlebars.c/issues/55

### CODE REVIEW

**Comment by rvalle** [comment]
@jirutka do you need anything else from me or contributor @jbboehr?

**Comment by rvalle** [comment]
@jirutka any chance someone will look at this PR?
this is my first contribution, i wonder if there is something else I should do.

File: testing/handlebars/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+_gitrev=8ea5f7163281e29e078cedf28801c7fb72493ed5
+_mustachespec_rev="83b0721610a4e11832e83df19c73ace3289972b9"
+_handlebarsspec_rev="e553716c186961477983d20c6e852d9a387f0668"
+
+depends="talloc"
+makedepends="automake autoconf libtool talloc-dev json-c-dev bison flex yaml-dev lmdb-dev pcre-dev check-dev"
+subpackages="$pkgname-dev $pkgname-utils"
+
+source="handlebars.c-${_gitrev}.tar.gz::https://github.com/jbboehr/handlebars.c/archive/${_gitrev}.tar.gz
```
Comment (jirutka) [inline testing/handlebars/APKBUILD]: Why do you use _gitrev instead of pkgver? https://github.com/jbboehr/handlebars.c/archive/v0.6.3.tar.gz

Comment (rvalle) [inline testing/handlebars/APKBUILD]: good point, its probably easier to maintain using the tag, give me a sec.

Comment (rvalle) [inline testing/handlebars/APKBUILD]: @jirutka , is this now ok?

**Comment by rvalle** [comment]
@jirutka did you have time to review the changes? is it OK now?

File: testing/handlebars/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+pkgrel=0
+pkgdesc="C implementation library of handlebars.js"
+url="https://github.com/jbboehr/handlebars.c"
+license="LGPLv2.1+"
+arch="all"
+
+_mustachespec_rev="83b0721610a4e11832e83df19c73ace3289972b9"
+_handlebarsspec_rev="e553716c186961477983d20c6e852d9a387f0668"
+
+depends="talloc"
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: removed, tested

File: testing/handlebars/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+makedepends="automake autoconf libtool talloc-dev json-c-dev bison flex yaml-dev lmdb-dev pcre-dev check-dev"
+subpackages="$pkgname-dev $pkgname-utils"
+
+source="handlebars.c-${pkgver}.tar.gz::https://github.com/jbboehr/handlebars.c/archive/v${pkgver}.tar.gz
+        spec-${_mustachespec_rev}.tar.gz::https://github.com/mustache/spec/archive/${_mustachespec_rev}.tar.gz
+        handlebars-spec-${_handlebarsspec_rev}.tar.gz::https://github.com/jbboehr/handlebars-spec/archive/${_handlebarsspec_rev}.tar.gz"
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: changed for mustache, requested tag for handlebars...

File: testing/handlebars/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+builddir="${srcdir}/handlebars.c-${pkgver}"
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: removed curly braces....

File: testing/handlebars/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+prepare() {
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: calling default_prepare

File: testing/handlebars/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    rm -rf ${builddir}/spec
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: fixed

File: testing/handlebars/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    mkdir ${builddir}/spec
+    ln -s ${srcdir}/spec-${_mustachespec_rev} ${builddir}/spec/mustache
+    ln -s ${srcdir}/handlebars-spec-${_handlebarsspec_rev} ${builddir}/spec/handlebars
+    cd "${builddir}"
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: removed redundant cd

File: testing/handlebars/APKBUILD [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+package() {
+	cd ${builddir}
+	make DESTDIR="${pkgdir}" install 
+}
+
+utils() {
+	pkgdesc="Handlebars command line utilities"
+        depends="handlebars"
```
Comment (rvalle) [inline testing/handlebars/APKBUILD]: > all tabs now...

Comment (rvalle) [inline testing/handlebars/APKBUILD]: removed auto-detected dependency

**Comment by rvalle** [comment]
@jirutka all the changed have been implemented, can you review them?

**Comment by rvalle** [comment]
@jirutka any chance you can review the changes?

**Pull request closed without merging**